app/internal/cmd/rules_read_exfil_refusal_test.go

```
@@ -0,0 +1,157 @@
+	keys := config.CompatEnvironmentVariables()
+	keys = append(keys,
+		"SOCKGUARD_INSECURE_ALLOW_READ_EXFILTRATION",
+		"SOCKGUARD_INSECURE_ALLOW_BODY_BLIND_WRITES",
+	)
+	for _, key := range keys {
+		value, ok := os.LookupEnv(key)
+		if !ok {
+			continue
+		}
+		t.Setenv(key, value)
+		os.Unsetenv(key)
```

golangci-lint (2.12.2)

[error] Error return value of `os.Unsetenv` is not checked

(errcheck)